x-pack/plugins/security_solution/common/detection_engine/get_query_filter.test.ts

```diff
@@ -363,49 +363,151 @@ describe('get_filter', () => {
         bool: {
           filter: [
             { bool: { minimum_should_match: 1, should: [{ match: { 'host.name': 'linux' } }] } },
+          ],
+          must: [],
+          must_not: [
             {
               bool: {
-                must_not: {
-                  bool: {
-                    filter: [
-                      {
-                        nested: {
-                          path: 'some.parentField',
-                          query: {
-                            bool: {
-                              minimum_should_match: 1,
-                              should: [
-                                {
-                                  match_phrase: {
-                                    'some.parentField.nested.field': 'some value',
+                should: [
+                  {
+                    bool: {
+                      filter: [
+                        {
+                          nested: {
+                            path: 'some.parentField',
+                            query: {
+                              bool: {
+                                minimum_should_match: 1,
+                                should: [
+                                  {
+                                    match_phrase: {
+                                      'some.parentField.nested.field': 'some value',
+                                    },
                                   },
+                                ],
+                              },
+                            },
+                            score_mode: 'none',
+                          },
+                        },
+                        {
+                          bool: {
+                            minimum_should_match: 1,
+                            should: [
+                              {
+                                match_phrase: {
+                                  'some.not.nested.field': 'some value',
                                 },
-                              ],
+                              },
+                            ],
+                          },
+                        },
+                      ],
+                    },
+                  },
+                ],
+              },
+            },
+          ],
+          should: [],
+        },
+      });
+    });
+
+    test('it should work with a list with multiple items', () => {
```

Comment:
nit: I think this test case could have a more descriptive title. Something like, "exception list with multiple exception items with the same entry".